### DISCUSSION TITLE
WIBNI there was the ability to search by group name?

### DISCUSSION THREAD

**Discussion by John-L** [code4recovery/12-step-meeting-list#1135] [Ideas]
Perhaps you'd consider adding the ability to find meetings given the group name, or a portion of the name.

Thanks. Great plug-in.

**Comment by joshreisner** [comment]
seems to be working for me! 

https://code4recovery.org/meetings?tsml-day=3&tsml-query=test
https://code4recovery.org/meetings?tsml-day=3&tsml-query=tes

could you share the example that's not working for you?

**Reply by John-L** [reply]
Well, it's a good news/ bad news situation. I didn't try it (it obviously works), and I have a multi-word search e.g. "the Solution" (quotes included), or potentially the+Solution as search strings.

Thanks in advance.

**Reply by John-L** [reply]
And if you're in there, maybe a hover-over tooltip explaining it all.

Thanks again.

**Reply by joshreisner** [reply]
i'm not quite sure what you mean, but for a more advanced search experience, maybe try the TSML UI appearance? that should have quote searching and live updating

**Reply by anchovie91471** [reply]
Are you referring to finding a meeting on the front end or in the admin?

**Reply by anchovie91471** [reply]
Can you send screenshots of what page you're referring to?

**Reply by John-L** [reply]
![Screenshot_20230517-114834](https://github.com/code4recovery/12-step-meeting-list/assets/7512058/ed0d460d-f6e5-4536-ba2b-45edc1b30681)
![Screenshot_20230517-114750](https://github.com/code4recovery/12-step-meeting-list/assets/7512058/21e27a12-8fd4-4d15-91e1-64d91b387cc9)

**Reply by John-L** [reply]
The front-end, and in this situation, I'm a user and don't believe I can change the UI.

**Reply by joshreisner** [reply]
here's our demo of TSML UI : https://code4recovery.org/tsml-ui

here's that example with quote-searching: https://code4recovery.org/tsml-ui?search=%22the+solution%22

TSML doesn't support search operators like quotes and +

**Reply by John-L** [reply]
My Area's web-site chooses not to use TSML. Can the "regular" UI be improved?

**Reply by joshreisner** [reply]
ive been focusing my effort on TSML UI since it supports more sites than only those running WordPress. but yes if we can find a PHP developer who has the bandwidth, then TSML search can be enhanced to support quotes

out of curiosity, do you know the Area's rationale for not wanting to switch?
